my-app/src/i18n/config.ts

```diff
@@ -0,0 +1,18 @@
+import i18n from 'i18next';
+import translationEn from './en/translationEn.json';
+import translationRu from './ru/translationRu.json';
+import { initReactI18next } from 'react-i18next';
+
+export const resources = {
+  en: {
```

Comment:
Maybe you can create a enum for languages and use it here like
```
const resources = {
  [Language.En]: {},
  [Language.Ru]: {},
}

...

i18n.use(initReactI18next).init({
  lng: Language.En',
  resources,
})
```
And also use enum to provide a value for languages in NavBar